quickbooks_desktop_endpoint.rb

```diff
@@ -234,7 +234,10 @@ def add_flow_return_payload
 
   def generate_and_add_guid
     @return_payload ||= {}
-    @return_payload[:external_guid] = "{#{SecureRandom.uuid.upcase}}"
+    guid = "{#{SecureRandom.uuid.upcase}}"
+
+    @payload[object_type][:external_guid] = guid
+    @return_payload[:external_guid] = guid
   end
```

Comment:
I'm guessing
`@return_payload[:external_guid] = @payload[object_type][:external_guid] = "{#{SecureRandom.uuid.upcase}}"`
wouldn't be as fun?

Reply:
Haha - Seems like a long line to me. Plus, I thought setting a guid variably and then explicitly setting it on each object was clearer to what was happening.